src/sentry/seer/endpoints/project_autofix_automation_settings.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+from typing import Any
+
+import sentry_sdk
+from django.db import router, transaction
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import ProjectPermission
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models.options.project_option import ProjectOption
+from sentry.models.project import Project
+from sentry.seer.autofix.constants import AutofixAutomationTuningSettings
+from sentry.seer.autofix.utils import (
+    AutofixStoppingPoint,
+    bulk_set_project_preferences,
+    get_project_seer_preferences,
+)
+from sentry.seer.models import SeerApiResponseValidationError
+
+
+class SeerAutofixSettingPutSerializer(serializers.Serializer):
+    """Serializer for OrganizationAutofixAutomationSettingsEndpoint.put"""
+
+    autofixAutomationTuning = serializers.ChoiceField(
+        choices=[opt.value for opt in AutofixAutomationTuningSettings],
+        required=False,
+        help_text="The tuning setting for the projects.",
+    )
+    automatedRunStoppingPoint = serializers.ChoiceField(
+        choices=[opt.value for opt in AutofixStoppingPoint],
+        required=False,
+        help_text="The stopping point for the projects.",
+    )
+
+    def validate(self, data):
+        if "autofixAutomationTuning" not in data and "automatedRunStoppingPoint" not in data:
+            raise serializers.ValidationError(
+                "At least one of 'autofixAutomationTuning' or 'automatedRunStoppingPoint' must be provided."
+            )
+        return data
+
+
+@region_silo_endpoint
+class ProjectAutofixAutomationSettingsEndpoint(ProjectEndpoint):
+    """Bulk endpoint for managing project level autofix automation settings."""
+
+    owner = ApiOwner.CODING_WORKFLOWS
+
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+        "PUT": ApiPublishStatus.PRIVATE,
+    }
+
+    permission_classes = (ProjectPermission,)
```

Comment:
this is not necessary as the ProjectEndpoint will already have this set https://github.com/getsentry/sentry/blob/74a33eb749ab7858561c58c927ca7770710f800b/src/sentry/api/bases/project.py#L130

Reply:
gone